libs/component-configuration/src/lib/component/component-configuration.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'rooi-component-config',
+    templateUrl: './component-configuration.component.html',
+    styleUrls: ['./component-configuration.component.scss']
+})
+export class ComponentConfigurationComponent {
+    @Input() title: string | undefined;
```

Comment:
for some reason ESLint wants these `| undefined` types. I don't like it though, should I update the eslint rules?